worlds/banjo_tooie/Items.py

```diff
@@ -0,0 +1,298 @@
+from BaseClasses import Item, ItemClassification
+from typing import Dict, NamedTuple
+from .Names import itemName, locationName
+
+
+class BanjoTooieItem(Item):
+    # 1230924 (Beans) but beware of level access keys that are way higher!
+    game: str = "Banjo-Tooie"
+
+
+class ItemData(NamedTuple):
+    btid: int | None = None
+    qty: int = 0
+    type: ItemClassification = ItemClassification.progression
+    default_location: str = ""
```

Comment:
These are always set anyway, so they don't really need to be there as defaults. Also it might be better to make `default_location` None when there isn't one as opposed to an empty string.